lib/exchange.js

```diff
@@ -101,18 +101,24 @@ const exchange = (name, type, exchangeOptions) => {
         publish(msg, opts);
     };
 
-    const rpcServer = (key, handler) => {
+    const rpcServer = (key, handler,options) => {
 
         if (!replyQueue) {
             throw new Error('replyQueue not set - ensure { noReply: false } is passed to exchange options');
         }
 
         const rpcQueue = createQueue({
-            key,
+          ...{
+            key: key,
             name: key,
             prefetch: 1,
             durable: false,
-            autoDelete: true
+            autoDelete: false,
+            arguments: {
+              'x-message-ttl': 30000
```

Comment:
can we use something like this:
`options?.x-message-ttl || 30000`
to have ability override it ...

Reply:
I added an option to pass options that will override the defaults.
note that overriding a queue option is not possible and we must delete the queue first

Reply:
So to override this value I need pass 
options.arguments.x-message-ttl
?